code/modules/research/designs/autolathe_designs.dm

```diff
@@ -949,3 +949,11 @@
 	materials = list(MAT_METAL = 1500, MAT_GLASS = 500)
 	build_path = /obj/item/mail_scanner
 	category = list("initial", "Miscellaneous")
+
+/datum/design/spraybottle
+	name = "Spray bottle"
+	id = "spraybottle"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 2000)
+	build_path = /obj/item/reagent_containers/spray
```

Comment:
What kind of spray you want it to print? Because this base path is not being utilised anywhere and its one that has 3 tiles range/250 units capacity.
I would suggest you testing it too because i think the range/consumption of it is not correct either.

Reply:
It should be simple spray bottle, like space cleaner's one, but empty. It prints it, sprite is correct, name too. Not tested range and capacity. I'll test it a bit later.

Reply:
This is the base path for all items that have the spray behavior, you would have to create a version of the space cleaner where the only difference is that it comes empty.

Reply:
It is the normal spray bottle, working fine, with range 3 and 250u capacity. Tested.
So... Why now cleaners have range 2???

Reply:
It was changed to a range of 2.
Another thing that was changed with cleaners is the amount of units that are used on the alternate modes too, which is probably different on that base path.

Reply:
1) That's just dumb, (I mean... Really, why?) but ok.
2)No problem there. Noting changed, actually. Yeah, nubers are different. But it ony matters if it starts on 10u per use or it is 5u. On both (/obj/item/reagent_containers/spray) and (/obj/item/reagent_containers/spray/cleaner) 10u mode spends 10u and 5u mode spends 5u

Reply:
Yeah, just make a subtype of the space cleaner spray bottle that doesn't have space cleaner and it'll be good

Reply:
> Yeah, just make a subtype of the space cleaner spray bottle that doesn't have space cleaner and it'll be good

It's not the right way. General "spray bottle" exists already, and creating new thing... Not good style.

Reply:
Just make it so it's mechanically identical to the normal space cleaner bottle, and yeah making a subtype from the base spray bottle is prob better

Reply:
Okay. Made changes